ruby/engine/measure_manager_server.rb

```diff
@@ -119,7 +119,7 @@ def do_POST (request, response)
 
         result = {}
 
-        data = JSON.parse(request.body, {:symbolize_names=>true})
+        data = JSON.parse(request.body || "{}", {:symbolize_names=>true})
```

Comment:
Minor tweaks in ruby CLI. When body is empty, you don't want to get bad error on JSON.parse with an ugly backtrace and `No implicit conversion of nil into String` isn't a good error message